server/src/test/java/io/crate/BlobHeadRequestHandlerTests.java

```diff
@@ -45,15 +33,62 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import org.elasticsearch.Version;
+import org.elasticsearch.cluster.node.DiscoveryNode;
+import org.elasticsearch.cluster.service.ClusterService;
+import org.elasticsearch.common.util.concurrent.EsExecutors;
+import org.elasticsearch.test.ClusterServiceUtils;
+import org.elasticsearch.test.ESTestCase;
+import org.elasticsearch.test.transport.CapturingTransport;
+import org.elasticsearch.threadpool.TestThreadPool;
+import org.elasticsearch.threadpool.ThreadPool;
+import org.elasticsearch.transport.TransportRequest;
+import org.elasticsearch.transport.TransportResponse;
+import org.elasticsearch.transport.TransportService;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.crate.blob.BlobTransferTarget;
+import io.crate.blob.DigestBlob;
+import io.crate.blob.transfer.HeadChunkFileTooSmallException;
+import io.crate.blob.transfer.PutHeadChunkRunnable;
 
 public class BlobHeadRequestHandlerTests extends ESTestCase {
 
+    private ThreadPool threadPool;
+    private CapturingTransport transport;
+    private ClusterService clusterService;
+    private TransportService transportService;
+
+    @Before
+    public void setUpResources() throws Exception {
+        threadPool = new TestThreadPool(getClass().getName());
+        transport = new CapturingTransport() {
```

Comment:
Is this used? I did not find any usage of it

Reply:
Couple lines below:

`transportService = transport.createTransportService(`

but could turn it into a local variable

Reply:
thx. read it multiple times but couldn't find it 🤦 
local var sounds good.